src/components/UI/use-toast.js

```diff
@@ -4,6 +4,7 @@ import * as React from 'react'
 const TOAST_LIMIT = 1
 const TOAST_REMOVE_DELAY = 1000000
 
+// eslint-disable-next-line no-unused-vars

```

Comment:
leave a comment what it will be used for

Reply:
checked after removing the code lines. It is working fine.